Assets/MixedRealityToolkit/_Core/Definitions/AxisType.cs

```diff
@@ -30,10 +30,18 @@ public enum AxisType
         /// </summary>
         DualAxis,
         /// <summary>
-        /// Dual Axis analogue input.
+        /// Position OR Rotation Axis analogue input. <-- Experiment
         /// </summary>
         ThreeDoF,
```

Comment:
Let's go ahead and remove this one.

Reply:
Rofl, I was going to remove the two new ones as I found a way around it following our discussion.

So, which method do you prefer??

* A single 3DoF
* Separate Position and Rotation 3DoF

#Fight

Reply:
> Separate Position and Rotation 3DoF